ragstack/colbert/colbert_embedding.py

```diff
@@ -25,33 +25,24 @@
 from .token_embedding import EmbeddedChunk, TokenEmbeddings
 
 
-def calculate_query_maxlen(tokens: List[List[str]], min_num: int, max_num: int) -> int:
+def calculate_query_maxlen(tokens: List[List[str]]) -> int:
```

Comment:
also need to update the call where this is used:

https://github.com/datastax/ragstack-ai/blob/main/ragstack/colbert/colbert_embedding.py#L224